### PR TITLE
Add utility method to retrieve velocity discontinuities

### LINKED ISSUE #9: Explicit discontinuities?
Hi!

I'm considering writing some native (seismic) ray tracing code in Julia, but I'll need to make sure that I'm sampling critical points like discontinuities. I could calculate these using this package as is (using some sort of finite difference scheme?) but I was wondering if it could instead be of interest to add explicit (hard-coded?) discontinuity estimates to the built-in models. Is it something you'd consider adding or would you rather keep such derived information outside this package?

Cheers

### PR DESCRIPTION
**Pull request by adigitoleo**
Closes #9

Adds a method to get velocity discontinuities from `LinearLayeredModel`s (other models can be converted using the constructor methods)

### CODE REVIEW

File: src/basic_properties.jl [hunk 1]
```diff
@@ -89,3 +89,25 @@ function voigt_velocities(vpv, vsv, vph, vsh, η)
...
+
+"""
+    discontinuities(m::LinearLayeredModel; depths=false)
+
+Return the radii (or depths if `depths=true`) of velocity discontinuities
+for the 1D `LinearLayeredModel` `m`.
+
+"""
+function discontinuities(m::LinearLayeredModel; depths=false)
+    @assert issorted(m.r)
```
Comment (anowacki) [inline src/basic_properties.jl:101]: The radii are checked for being sorted (not as elegantly as this) in the constructors, so this shouldn't be needed unless someone has been messing with the internals of the model.

**Comment by anowacki** [comment]
Thanks very much for this!  I'm very happy to have this in and I really appreciate you submitting your improvements.

As you prefer, either you or I can do the following and then I'll merge:
1. Remove the `@assert` in `discontinuities`.
2. Add `discontinuities` to the function index in the docs.

I do wonder whether the indices of the discontinuities might also be useful to people, since these have been calculated in the function anyway, so I'd be interested in your opinion on that.

**Comment by adigitoleo** [comment]
I could return the indices too, I guess in a `Tuple`. Do we want to return the index of the first or second of the repeated radii (`i-1` or `i` in the implementation)?

**Comment by adigitoleo** [comment]
I could return the indices too, I guess in a `Tuple`. Do we want to return the index of the first or second of the repeated radii (`i-1` or `i` in the implementation)?

**Comment by anowacki** [comment]
I agree that the choice is arbitrary, but I'm happy with your choice nonetheless.  I've merged this and will add that to the docstring.

Thanks again for the contribution!  I'll bump the minor version to reflect the change when I've updated the CI workflow as it's now out of date.

**Pull request merged**